plugin/web/sharedComponents/Icon/icons.ts

```diff
@@ -61,4 +62,5 @@ export default {
   empty: Empty,
   newPage: NewPage,
   edit: Edit,
+  sun: Sun,
```

Comment:
Can you rename this a bit better? The icon definitely isn't a sun. Maybe `target` or `bullseye`?

Reply:
yeah done

Reply:
fc8de4437577d111f90b0a1c7db09d37c48d4de6